### DISCUSSION TITLE
Publish documentation as pdf

### DISCUSSION THREAD

**Discussion by ArchKudo** [nextflow-io/nextflow#4392] [Ideas]
Hi,
I am new to Nextflow and prefer learning from physical books rather than a screen.
But, I couldn't find a copy of the documentation as a pdf to print it.
I even tried cloning the repo and generating a latex pdf from sphinx docs, but it complains about mermaid diagrams, svg, and Unicode glyphs which seem to be missing from the resulting pdf.
Was wondering if there was an official pdf documentation or, any guidance on resolving the above errors would be equally helpful.

__Steps to reproduce__:

```sh
sudo apt install texlive-full
cd ~/workplace/nextflow/docs
python -m venv .venv
source .venv/bin/activate
pip install -r requirements.txt
make clean latexpdf
```

__Host Details__:

```sh
.venv ❯ uname -a
Linux wbar 5.15.90.1-microsoft-standard-WSL2 #1 SMP Fri Jan 27 02:56:13 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
.venv ❯ python --version
Python 3.11.4
```
[Build Output Gist](https://gist.githubusercontent.com/ArchKudo/a3a223f5b518f6e6a80a0285ce5f6607/raw/e7e2a4ea6d006d828871562f31d2c350db6ac05b/logs)

**Comment by bentsherman** [comment]
We haven't really supported the latex output since no one uses it (until now). We recently added some mermaid diagrams to the docs which are rendered at build time. It looks like something is missing to make this part work with latex.

The plugin we use is [sphinxcontrib-mermaid](https://github.com/mgaitan/sphinxcontrib-mermaid), maybe you can adjust the `conf.py` to make it work. Happy to merge the necessary changes if you get it to work.
